core/src/test/java/com/yahoo/ycsb/TestUtils.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2016 Yahoo! Inc. All rights reserved.
```

Comment:
Instead of Yahoo! Inc. This should be `Copyright (c) 2016 YCSB contributors. All rights reserved.`